blog/_posts/2017-09-06-gsoc-derivative_operators.md

```diff
@@ -0,0 +1,156 @@
+---
+layout: post
+title:  "GSoC 2017: Efficient Discretizations of PDE Operators"
+author: Shivin Srivastava, Christopher Rackauckas
+---
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS_HTML"></script>
+
+<script type="text/x-mathjax-config">
+MathJax.Hub.Config({
+tex2jax: {
+inlineMath: [ ['$','$'], ["\\(","\\)"] ],
+displayMath: [ ['$$','$$'], ["\\[","\\]"] ],
+processEscapes: true,
+processEnvironments: true
+},
+// Center justify equations in code and markdown cells. Elsewhere
+// we use CSS to left justify single line equations in code cells.
+displayAlign: 'center',
+"HTML-CSS": {
+styles: {'.MathJax_Display': {"margin": 0}},
+linebreaks: { automatic: true }
+}
+});
+</script>
+
+This project is an attempt towards building a PDE solver for JuliaDiffEq using the [Finite Difference Method](https://en.wikipedia.org/wiki/Finite_difference_method)(FDM) approach. We take up the FDM approach instead of [FEM](https://en.wikipedia.org/wiki/Finite_element_method) and [FVM](https://en.wikipedia.org/wiki/Finite_volume_method) as there are many toolboxes which already exist for FEM and FVM but not for FDM. Also, there are many use cases where the geometry of the problem is simple enough to be solved by FDM methods which are much faster due to their being able to avoid the bottleneck step of matrix multiplication by using Linear transformations to mimic the effect of a matrix multiplication.
```

Comment:
"by using Linear transformations" I don't think that's correct. All matrix operations are linear transformations. The key here is avoiding large sparse matrices and using matrices with more structure.

Reply:
I meant to say that we are avoiding matrix multiplication by using a function to modify the vector which mimics the effect of matrix multiplication.

Reply:
Yes, say it more directly.